### PR TITLE
feat: display profile picture

### LINKED ISSUE #162: Trying to fetch profile picture from microsoft IDP results in 500 from helix admin
Auth info [seems to be missing](https://github.com/adobe/helix-admin/blob/main/src/profile/handler.js#L40)

### LINKED ISSUE #347: Display profile picture

### PR DESCRIPTION
**Pull request by rofe**
Fix #347 
Fix #162

### CODE REVIEW

File: src/extension/app/store/app.js [hunk 1]
```diff
@@ -730,6 +730,7 @@ export class AppStore {
         'deleted',
         'logged-in',
         'logged-out',
+        'hidden',
```
Comment (rofe) [inline src/extension/app/store/app.js:733]: Forgot to add this back when we added the close button.

File: src/extension/app/components/plugin/plugin-action-bar.js [hunk 2]
```diff
@@ -430,22 +430,13 @@ export class PluginActionBar extends ConnectedElement {
...
-    }
-
-    if (value === 'project-added' || value === 'project-removed') {
+    } else if (value === 'project-added' || value === 'project-removed') {
       this.appStore.sampleRUM('click', { source: 'sidekick', target: value });
       chrome.runtime.sendMessage({ action: 'addRemoveProject' });
-      return;
     }
-
-    this.appStore.fireEvent(value);
```
Comment (rofe) [inline src/extension/app/components/plugin/plugin-action-bar.js:448]: Not part of the public extension API, sampleRUM is enough.

File: src/extension/app/store/app.js [hunk 3]
```diff
@@ -1332,8 +1333,6 @@ export class AppStore {
           this.setupPlugins();
           this.fetchStatus();
           this.fireEvent(EXTERNAL_EVENTS.LOGGED_OUT, this.status.profile);
-          // refresh the page to reflect logged out state
-          this.reloadPage();
```
Comment (rofe) [inline src/extension/app/store/app.js:1336]: I don't see why this should be required. We don't reload the page on login, either.

**Review by dylandepass — APPROVED** [review]

**Pull request merged**